_posts/2022-04-13-stepping-out-of-the-knowledge-zone.md

```diff
@@ -0,0 +1,177 @@
+experiences with such aims and how it was not working as smoothly as imagined.
+
+The second week in the velocity team also brought along some team building time. I joined a coffee call and it was
+super interesting to finally also feel like I was getting to know the other mysterious team members of velocity. We
+talked about our international backgrounds and it was great to hear so many stories about where people had come
+from and what languages they had grown up with. This finally was melting my heart for the team and I lost my last
+little bit of fear and shyness.
+
+As I then started to look at integrity and list some of the tests in it that were flakey Joey, our director of
+engineering, was helping me out leaving meaningful hints and comments on my notes. This plus the recent successful
+coffee call with the team encouraged me to my next very risky step: In a meeting called "velocity mob session" I
+proposed to the team members to mob together on some problem of integrity actually not being able to run certain
+tests from two instances in the same time. It felt risky and scary driving a mob session in a repository that I
+wasn't familiar with, reproducing a problem that I wasn't fully understanding and trying to solve it in a
+language/framework that I was not an expert in. But the risk was totally worth it. It turned out to be a super
+interesting and fun and collaborative mob session and we actually figured stuff out together with the whole team.
+Nothing feels better than having a whole team figuring something out together. It really bonds to see stuff
+happening as a shared experience.
+
+The third week started and in a standup I admitted that I had not been following up on implementing a new role into
+our _old-admin_ because I felt uneasy to add something new to a project that was stuck in a weird mid-deprecation
+state. The team reacted quickly with a super good idea: Instead of adding something to the _old-admin_ I should set
+up something that was going to be the new-admin in our environment so that people could contribute to it on the
+upcoming hackathon. I was thrilled! Never have I witnessed the birth of a new project at Artsy and now I was
+supposed to set the whole thing up and go through each step of putting it out there? Awesome! Also I had absolutely
+no idea what that meant. Did I need to do some dangerous mystic exploration into the far away field of our system
+being set up in kubernetes or something? I wasn't sure. All I had in mind were these complex huge graphs of our
+system and how all the services were connected. How could I add another service into this huge spider web?
+
+The first helpful thing was a little check list that Joey gave me of things that had to be done. Breaking the task
+down into smaller task definitely already demystified the whole thing a bit. It looked something like:
+
+- get typical local development working
+- get `hokusai dev start` working (i.e., local development on docker)
```

Comment:
What do you think about adding a link to the `hokusai` and/or a brief description what this project represents?

Reply:
Good idea!